polkadot/xcm/xcm-builder/src/pay.rs

```diff
@@ -71,8 +71,8 @@ impl<
 		Timeout: Get<Querier::BlockNumber>,
 		Beneficiary: Clone,
 		AssetKind,
-		AssetKindToLocatableAsset: Convert<AssetKind, LocatableAssetId>,
-		BeneficiaryRefToLocation: for<'a> Convert<&'a Beneficiary, MultiLocation>,
+		AssetKindToLocatableAsset: TryConvert<AssetKind, LocatableAssetId>,
+		BeneficiaryRefToLocation: for<'a> TryConvert<&'a Beneficiary, MultiLocation>,
```

Comment:
there is a separate PR for this - https://github.com/paritytech/polkadot-sdk/pull/1572